Veriff/Modules/Passport/View/PassportViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  PassportViewController.swift
+//  Veriff
+//
+//  Created by Kseniya Zharikova on 27/7/22.
+//
+
+import Foundation
+import UIKit
+
+final class PassportViewController: UIViewController {
+    
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 20, weight: .medium)
+        label.textColor = .white
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.text = "To finish with verification we need a photo of your document"
+        view.addSubview(label)
+        return label
+    }()
+    
+    private lazy var descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 16)
+        label.textColor = .white
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.text = "Please take a photo of the front side of your ID-card. Make sure it is clearly visible."
+        view.addSubview(label)
+        return label
+    }()
+    
+    private lazy var takePhotoButton: UIButton = {
+        let button = UIButton()
+        button.backgroundColor = .white
+        button.setTitle("Take a photo", for: .normal)
+        button.setTitleColor(UIColor.gray, for: .normal)
+        button.layer.cornerRadius = 8
+        button.addTarget(self, action: #selector(takePhotoAction(_:)), for: .allEvents)
+        view.addSubview(button)
+        return button
+    }()
+    
+    var presenter: PassportPresenter!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpUI()
+    }
+    
+    func setUpUI() {
```

Comment:
```suggestion
    func setupUI() {
```

setup в данном случае одно слово